include/singa/io/writer.h

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
update the comment to be consistent with other C++ code,
replace /*...*/ to ///, remove '@param'